src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/PublisherFunction.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Model;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Services.Interfaces;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+
+namespace GovUk.Education.ExploreEducationStatistics.Publisher.Functions
+{
+    public class PublisherFunction
+    {
+        private readonly IContentCacheGenerationService _contentCacheGenerationService;
+        private readonly IPublishingService _publishingService;
+        private readonly IReleaseInfoService _releaseInfoService;
+
+        public PublisherFunction(IContentCacheGenerationService contentCacheGenerationService,
+            IPublishingService publishingService,
+            IReleaseInfoService releaseInfoService)
+        {
+            _contentCacheGenerationService = contentCacheGenerationService;
+            _publishingService = publishingService;
+            _releaseInfoService = releaseInfoService;
+        }
+
+        [FunctionName("GenerateReleaseContent")]
+        public void GenerateReleaseContent(
+            [QueueTrigger("generate-release-content")]
+            GenerateReleaseContentMessage message,
+            ExecutionContext executionContext,
+            ILogger logger)
+        {
+            logger.LogInformation($"{executionContext.FunctionName} triggered: {message}");
+            _contentCacheGenerationService.GenerateReleaseContent(message).Wait();
+            logger.LogInformation($"{executionContext.FunctionName} completed");
+        }
+
+        [FunctionName("PublishReleaseContent")]
+        public void PublishReleaseContent([TimerTrigger("0 30 9 * * *")] TimerInfo timer,
+            ExecutionContext executionContext,
+            ILogger logger)
+        {
+            logger.LogInformation($"{executionContext.FunctionName} triggered at: {DateTime.Now}");
+            // TODO EES-865 Move content daily at 09:30
```

Comment:
Is this TODO not achieved with the cron expression above?

Reply:
It was just there because the function currently doesn't do anything at all. I've changed it to remove the bit about 09:30.